code/game/objects/items.dm

```diff
@@ -26,6 +26,8 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 
 	max_integrity = 200
 
+	attackable = FALSE

```

Comment:
Items are 100% attackable, what is this?

Reply:
nuh uh, you can attack items, you can't hit them though, even on harm intent

i can reword it to hittable if it makes more sense in this context